main.c

```diff
@@ -730,7 +733,7 @@ STATIC bool run_code_py(safe_mode_t safe_mode, bool *simulate_reset) {
 
 vstr_t *boot_output;
 
-STATIC void __attribute__ ((noinline)) run_boot_py(safe_mode_t safe_mode) {
```

Comment:
Why remove this attribute? It doesn't seem related to this change.

Reply:
I have put it back.